staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission_test.go

```diff
@@ -131,10 +132,11 @@ func TestAdmit(t *testing.T) {
 	}
 
 	type test struct {
-		hookSource    fakeHookSource
-		path          string
-		expectAllow   bool
-		errorContains string
+		hookSource      fakeHookSource
+		path            string
+		expectAllow     bool
+		errorContains   string
+		expectStatusErr bool
```

Comment:
This seems redundant. We always expect it to be a status error.

Reply:
No, we don't. `callHook` may return `ErrCallingWebhook`

Reply:
Does ErrCallIingWebhook get translated into a status error anywhere? All errors must be turned into status errors by the time they get to the thingy that writes the HTTP response. I made a suggestion to address this above.

Reply:
We should maybe make a comment on the admission plugin interface declaring that it always must return status errors?